### PR TITLE
Remove redundant code.

### PR DESCRIPTION
**Pull request by luokuning**

### CODE REVIEW

File: lib/network.js [hunk 1]
```diff
@@ -139,16 +139,14 @@ function calcNetworkSpeed(iface, rx, tx, operstate) {
...
-    tx: 0,
+    rx,
+    tx,
     rx_sec: -1,
     tx_sec: -1,
     ms: 0
   };
 
   if (_network[iface] && _network[iface].ms) {
-    result.rx = rx;
```
Comment (ItsEcholot) [inline lib/network.js]: Why are these lines redundant?

**Comment by luokuning** [comment]
@ItsEcholot Because `result.rx` and `result.tx` will be assign to `rx` and `tx` regardless of the condition result of the `if` expression. So I think we can assign at the beginning.

**Comment by ItsEcholot** [comment]
@luokuning Makes sense, sorry my brain failed me there.

**Comment by luokuning** [comment]
@ItsEcholot No worries. Happy to clarify.

**Comment by sebhildebrandt** [comment]
Thank you so much for checking the code! I already made the changes and published an update.

**Pull request closed without merging**